designs/administrator/olm/List-of-operators-in-all-projects/list-of-operators.md

```diff
@@ -0,0 +1,41 @@
+
+---
+parent: Administrator
+version: 4.5
+---
+
+# List of Operators
+
+## All projects view
+
+- When a user has selected the All Projects view from the context selector on the Installed Operators page, here they can view all of their Installed Operators. The goal of this design is to simplify how a user views the namespaces and associated operator(s) when they have selected to view the Installed Operators under All Projects.
+
+- When a user clicks the link in the table, “View 8…”, an anchor link takes the user to the Namespaces in the body of the Operator Details page. In this flow, we opt for a filterable experience in which the user can later filter by namespace or project under tabs such as Provided APIs or Events. (Scroll to view experience)
+
+- In an alternate path, if a user were to click into the Operator itself, it would take the user to the top of the Operator Details page rather than the anchor link in the body of the page. 
+
+![list-of-operators](img/list-of-operators.jpg)
```

Comment:
Great point. I'm happy to mock that up.

Reply:
Thanks for making the change on the Installed Operators list view!
Could you also add a mockup for " _'Elasticsearch' (Operands) list view and viewing instances from 'all projects'_ " after users clicked on 'Elasticsearch' link?
(thinking it could be added perhaps after the "Events" section of this doc)

Something similar to this:
![operand_instances_list__all-projects_view](https://user-images.githubusercontent.com/5903705/76130240-08d83200-5fbf-11ea-8e8d-b0c6b5b09a44.png)

This is worth pointing out as it enables users to see all Operands across different Namespaces, thanks!

Reply:
Thanks for making the change on the Installed Operators list view!
Could you also add a mockup for " _'Elasticsearch' (Operands) list view and viewing instances from 'all projects'_ " after users clicked on 'Elasticsearch' link?
(thinking it could be added perhaps after the "Events" section of this doc)

Something similar to this:
![operand_instances_list__all-projects_view](https://user-images.githubusercontent.com/5903705/76130240-08d83200-5fbf-11ea-8e8d-b0c6b5b09a44.png)

This is worth pointing out as it enables users to see all Operands across different Namespaces, thanks!